README.md

```diff
@@ -24,13 +24,19 @@ Install the following tools if you don't have them already.
 carefully. In particular, configure your shell/environment so
 that the tools are available in every terminal/command window you create.
 
-### 2. Clone this repo
+### 2. Clone this repo _and_ its submodule
 
-1. **Create or choose a directory** to hold this site's Git repository, and the
-   other repositories needed to build this site (which will be fetched later);
-   for example, `~/git`.
-1. **Clone this repo** ([site-www][]) into the chosen directory by following
-   the instructions given in the GitHub help on [Cloning a repository][].
+> NOTE: This repo has a git _submodule_, which affects how you clone it.
+
+To **clone this repo** ([site-www][]), follow the instructions given in the
+GitHub help on [Cloning a repository][], and _choose one_ of the following
+submodule-cloning techniques:
+
+- Clone this repo and its submodule _at the same_: use the
+  `--recurse-submodules` option with the git clone command for this repo.
```

Comment:
Let's add the full command here, both as an example and as a quick default. It's unlikely that we'll change the path of where the repo lives anytime soon.

Reply:
Done